packages/fast-components-react-base/src/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Foundation, {HandledProps} from "../foundation";
+import {ICheckboxHandledProps, ICheckboxManagedClasses, ICheckboxUnhandledProps} from "./checkbox.props";
+import {ICheckboxClassNameContract, IManagedClasses} from "@microsoft/fast-components-class-name-contracts";
+import {get} from "lodash-es";
+
+/* tslint:disable-next-line */
+class Checkbox extends Foundation<ICheckboxHandledProps & ICheckboxManagedClasses, ICheckboxUnhandledProps, {}> {
+    protected handledProps: HandledProps<ICheckboxHandledProps & IManagedClasses<ICheckboxClassNameContract>> = {
+        managedClasses: void 0,
+        text: void 0,
+        checked: void 0,
+        indeterminate: void 0,
```

Comment:
I think you'll want to apply the indeterminate prop to the input

Reply:
+1 

From an implementation standpoint, @khamudom, does the indeterminate instance show up when passed? @nicholasrice - I know previously this had to be added in componentDidMount; hopefully that isn't the case anymore?

Reply:
indeterminate is now in the input